modules/sor/sorV2/lib/pools/gyro2/gyro2Pool.ts

```diff
@@ -229,4 +229,38 @@ export class Gyro2Pool implements BasePool {
 
         return { tIn, tOut, sqrtAlpha, sqrtBeta };
     }
+
+    addLiquiditySingleTokenExactIn(tokenIn: Token, bpt: Token, amount: TokenAmount): TokenAmount {
+        throw new Error('Gyro2 - addLiquiditiySingleTokenExactIn: Invalid Method');
```

Comment:
Ok, so following up on what we discussed at the on-site, we'll stick with only Weighted/Stable pools on this first iteration, right? So please remove all other pool types.
We'll still go with previous pool math structure for this PR, but the idea is to setup the V3 math structure on a following PR and use it for all V3 operations, since it should be easier to add new pool types.
Later on, we should be able to evaluate if it's ok to move everything to the new math structure as ideally it should yield the same result.